r/tests/testthat/test-duckdb.R

```diff
@@ -202,9 +202,9 @@ dbExecute(con, "PRAGMA threads=2")
 on.exit(dbDisconnect(con, shutdown = TRUE), add = TRUE)
 
 test_that("Joining, auto-cleanup enabled", {
-  # ARROW-17643: A change in duckdb 0.5.0 caused this test to fail but will
-  # be fixed in the next release.
-  skip_if_not(packageVersion("duckdb") > "0.5.0")
+  # ARROW-17643, ARROW-17818: A change in duckdb 0.5.0 caused this test to fail
+  # TODO: ARROW-17809 Follow up with the latest duckdb release to solve the issue
+  skip("ARROW-17818: Latest DuckDB causes this test to fail")
```

Comment:
I haven't been following this particular bug too closely so don't know this off the top of my head: is this the only test block that fails? And if so: does it fail during cleanup or during the query?

This is a pretty low-risk test to generically skip if this is the only test that is an issue since we have basically the same query without autocleanup below.

Reply:
That sounds a bit dismissive. Just because this particular test doesn't replicate it doesn't mean it may not occur more frequently with another sequence of operations.